### PR TITLE
Update angular docs and sample with new package

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR:
- Updates references to msal-browser and msal-angular to @azure/msal-browser and @azure/msal-angular, and 1.x and 2.x to v1 and v2
- Updates samples to use the new beta package
- Removes production warning in README

### CODE REVIEW

File: lib/msal-angular/docs/v2-docs/1.x-2.x-upgrade-guide.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
-# Upgrading from MSAL Angular 1.x to 2.x
+# Upgrading from MSAL Angular v1 to v2
 
-MSAL Angular 2.x brings our Angular wrapper up-to-date with the latest version of MSAL common, and with out-of-the-box support for modern versions of Angular (9 - 11) and rxjs (6).
+MSAL Angular v2 brings our Angular wrapper up-to-date with the latest version of MSAL common, and with out-of-the-box support for modern versions of Angular (9 - 11) and rxjs (6).
 
-This guide will demonstrate changes needed to migrate an existing application from `@azure/msal-angular@1.x` to `@azure/msal-angular@2.0.0-alpha.0`.
+This guide will demonstrate changes needed to migrate an existing application from `@azure/msal-angular@1.x` to `@azure/msal-angular@2.0.0-beta.0`.
```
Comment (tnorling) [inline lib/msal-angular/docs/v2-docs/1.x-2.x-upgrade-guide.md:5]: Small suggestion so you don't need to update this again after GA

```suggestion
This guide will demonstrate changes needed to migrate an existing application from `@azure/msal-angular` v1 to v2.
```

**Review by tnorling — APPROVED** [review]

File: lib/msal-angular/README.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ Note: `msal-angular@2` is under development. **We do not recommend using this ve
...
 
-This is an improvement upon the current `msal-angular` library which will utilize the authorization code flow. Most features available in the old library will be available in this one, but there are nuances to the authentication flow in both. The `@azure/msal-angular` package does NOT support the implicit flow.
+The current `@azure/msal-angular` library improves upon the previous version and utilizes the authorization code flow. Most features available in the old library will be available in this one, but there are nuances to the authentication flow in both. The latest `@azure/msal-angular` package does NOT support the implicit flow.
 
 ## Guides
```
Comment (jo-arroyo) [inline lib/msal-angular/README.md:35]: @jasonnutter Do these upgrade guides need to be changed to v1-v2 as well?

File: lib/msal-angular/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Microsoft Authentication Library for Angular
 
 
-Note: `msal-angular@2` is under development. **We do not recommend using this version in a production environment yet**.
+Note: MSAL Angular v2 is under development. **We do not recommend using this version in a production environment yet**.
```
Comment (jasonnutter) [inline lib/msal-angular/README.md:4]: For public preview we can remove the warning to note use in prod.

**Review by jasonnutter — APPROVED** [review]
One thing otherwise looks good!

File: lib/msal-angular/README.md (hunk 2 above)
Comment (jasonnutter) [reply lib/msal-angular/README.md:35]: Yeah probably a good idea to be explicit.

File: lib/msal-angular/README.md (hunk 3 above)
Comment (jo-arroyo) [reply lib/msal-angular/README.md]: Noted. Done

**Pull request merged**